scripts/configure

```diff
@@ -218,6 +218,8 @@ APP_SYNCTHING_PORT="8384"
 APP_SYNCTHING_SYNC_PORT="22000"
 APP_UPTIME_KUMA_PORT="8385"
 APP_UPTIME_KUMA_IP="10.21.21.62"
+APP_ITCHYSATS_IP="10.21.21.63"
```

Comment:
There is an IP collision with `$APP_RIDE_THE_LIGHTNING_BOLTZ_IP` in master here.

```suggestion
APP_ITCHYSATS_IP="10.21.21.64"
```

Reply:
👍 [`218b8dd` (#1149)](https://github.com/getumbrel/umbrel/pull/1149/commits/218b8dd021a8bea5817b3ee0e110fad98023f858)